spec/support/features/webauthn_helper.rb

```diff
@@ -37,9 +37,9 @@ def mock_press_button_on_hardware_key_on_setup
     set_hidden_field('attestation_object', attestation_object)
     set_hidden_field('client_data_json', setup_client_data_json)
 
-    button = first('#submit-button', visible: false)
+    button = first('#continue-button')
     if javascript_enabled?
-      button.execute_script('this.click()')
+      page.evaluate_script('document.querySelector("form").submit()')
```

Comment:
Because we have to skip over the browser prompt, to properly bypass that we need to just submit the form via javascript instead of through the UI when javascript is enabled.